### ISSUE TITLE
Reading parquet before saving them as parquet will cause FileNotFoundError

### ISSUE THREAD

**Issue by xinyuejohn** [USM-CHU-FGuyon/BlendedICU#21]
In 1_extract_mimic.py:

```
mimic_prep = mimicPreparator(
    chartevents_pth='/icu/chartevents.csv.gz',                         
    labevents_pth='/hosp/labevents.csv.gz')

mimic_prep.load_raw_tables()
```

`self.icustays = self._icustays()` was called in `mimicPreparator.__init__()`. And it tries to read parquets. However, parquets are only saved in the next step using `load_raw_tables()`

**Comment by USM-CHU-FGuyon** [comment]
Hi, Thanks for your PR. I will include this in the next version.

**Comment by USM-CHU-FGuyon** [comment]
This should be fixed in v0.3.0, thanks for the input !
